practices/lockfile.md

```diff
@@ -31,7 +31,7 @@ A practice in our DX Scanner called LockFileIsPresent verifies if your source re
 
 LockFileIsPresent is our DX Scanner practice that tells you if your source repository contains a lock file. Many people decide to ignore the lock files and they do not commit it to Git. What is a lock file and why should your repository contain one?
```

Comment:
Rewrite the Lockfile as mentioned earlier.

Reply:
https://github.com/DXHeroes/knowledge-base-content/pull/202#discussion_r334341275